.babelrc

```diff
@@ -2,7 +2,6 @@
   "plugins": [
     "version"
   ],
-  "presets": [["@babel/preset-env"]],
```

Comment:
Parcel includes its own babel plugins, and produces larger files if this is left in. For the parts of the build using Babel standalone, I add this back in via a command line argument.